src/components/SaveButton/SaveButton.jsx

```diff
@@ -30,11 +30,9 @@ const SaveButton = ({ className, type }) => {
     if (window.plausible) {
       window.plausible("Save button");
     }
-    if (!user) {
-      document.dispatchEvent(logInEvent);
-    }
+    document.dispatchEvent(logInEvent);
     dispatch(triggerSave());
-  }, [dispatch, user]);
+  }, [dispatch]);
```

Comment:
I'm slightly confused by this... Is this because we're now hiding the save button when someone is logged in? Or does nothing happen anyway if you try and dispatch a log in event if they're already logged in?

Reply:
so nothing happens editor side due to this custom event being triggered but host side it'll initiate the login flow however if that flows already been completed it'll just set the `user` object in state